src/metabase/models/setting.clj

```diff
@@ -122,7 +122,12 @@
   these names to avoid unintended side-effects if an application database still stores values for these settings."
   #{"-site-url"
     "enable-advanced-humanization"
-    "metabot-enabled"})
+    "metabot-enabled"
+    "ldap-sync-admin-group"})
+
+(def ^:dynamic *allow-retired-setting-names*
+  "Primarily used in test to disable retired setting check"
```

Comment:
https://guide.clojure.style/#docstring-summary

Reply:
The variable name isn't that confusing but a docstring should still describe what the variable/function does. Right now it just describes where it is used. For example if I had a function like

```clj
(defn my-function
  "This is mostly used in [[metabase.my-namespace]]."
  [x]
  ...)
```

That tells me nothing about what `my-function` does. This is better:

```clj
(defn my-function
  "`my-function` does blah blah blah. This is mostly used in [[metabase.my-namespace]]."
  [x]
  ...)
```

Reply:
Also please use complete sentences